src/scripts/settings.ts

```diff
@@ -14,4 +14,13 @@ export module Settings {
 
 		return localResult.Value;
 	}
+
+	export function setSettingsJsonForTesting(jsonObject?: {}): void {
```

Comment:
Probably should go in another change, but settings.ts should not expose a method like this where it can potentially get misused. If we want to test with custom settings, an alternative approach might be to use dependency injection, where in our test environments we can inject a different object, and in prod environments, we can inject the one in the settings file.